enzy_htp/_interface/pymol_interface.py

```diff
@@ -101,6 +101,35 @@ def select_pymol_obj(self, pattern: str, pymol_obj_name: str,
 
         return sorted(result)
 
+    def point_mutate(self, pos_key: Tuple[str, int], 
+                     target: str, pymol_obj_name: str, 
+                     pymol_session: pymol2.PyMOL) -> None:
+        """
+        Performs a single point mutation on the PDB file in the PyMOL session in-place.
+        Args:
+            pos_key: the chain id and residue index.
+            target: the target residue name (3 letters).
+            pymol_obj_name: the name of the PDB file in PyMOL.
+            pymol_session: the current PyMOL session.
+        Returns:
+            None.
+        """
+
+        # load pymol wizard mutagenesis function
+        pymol_session.cmd.wizard("mutagenesis")
+        pymol_session.cmd.do("refresh_wizard")
+        
+        # select res idx to mutate to target
+        pymol_session.cmd.get_wizard().set_mode(target)
+        if pymol_obj_name == "":
+            pymol_session.cmd.get_wizard().do_select(pos_key[0] + "/" + str(pos_key[1]) + "/")
```

Comment:
I suggest usingh f-string when handling string like this
https://realpython.com/python-f-strings/